frontend/src/app/main-table/cost-table/cost-table.component.html

```diff
@@ -5,23 +5,29 @@
   </div>
 
   <p>
-    Estimated cost per day of notebook server resources – summed values may not
+    Estimated cost of notebook server resources over the last 24 hours – summed values may not
```

Comment:
What's the reason for this change?

Reply:
@ca-scribner and I felt it was confusing to some users as it implied it was the cost of resources for the current day, so if a new resource was added, user might think the cost should update.

Reply:
yeah the estimate given here isn't a forward-looking per-day cost estimate (I don't think?), but instead a backward 24-hour assessment of what you used.  So if today I run nothing, then tomorrow I turn on a GPU and lots of expensive stuff and immediately look at kubecost, it'll still say "Estimated cost per day of notebook server resources = $0".  But tomorrow if I look again, it'll be $$$.  So the wording change was suggested to make it clear we're estimating based on history, not current running resources

Even better would be if we did our own forward-looking estimate based on a shorter timeframe if that's possible.  for example, do something like "last minute of costs * 24 * 60 * 60 = estimated cost per day based on your usage in the last minute"

...unless I misunderstood kubecost's output.  If it is forward looking already, then the old wording is great

Reply:
:+1:  `Estimated -> Approximate`